x-pack/plugins/apm/public/components/app/Settings/CustomizeUI/CustomLink/index.test.tsx

```diff
@@ -115,7 +115,6 @@ describe('CustomLink', () => {
       act(() => {
         fireEvent.click(getByText('Create custom link'));
       });
-      await wait(() => expect(callApmApiSpy).toHaveBeenCalled());
```

Comment:
Any idea why we used to have this line and now it's not longer needed?

Reply:
`callApmApiSpy` is called in the `Preview` section, it's not something we require to check to open the flyout that's why I removed it and made the `openFlyout` not async anymore.